scipy/signal/signaltools.py

```diff
@@ -404,15 +404,15 @@ def fftconvolve(in1, in2, mode="full", axes=None):
         in1, s1, in2, s2 = in2, s2, in1, s1
 
     # Speed up FFT by padding to optimal size for FFTPACK
-    fshape = [fftpack.helper.next_fast_len(d) for d in shape[axes]]
+    fshape = [sp_fft.next_fast_len(d) for d in shape[axes]]
     fslice = tuple([slice(sz) for sz in shape])
     # Pre-1.9 NumPy FFT routines are not threadsafe.  For older NumPys, make
```

Comment:
Since we're not using numpy anymore the threadsafe stuff can be removed?

Reply:
That's in #9876, I'm trying not to mix them too much